src/FormattingSettingsService.ts

```diff
@@ -29,7 +29,7 @@ export class FormattingSettingsService implements IFormattingSettingsService {
                 if (card instanceof CompositeCard) card.topLevelSlice?.setPropertiesValues(dataViewObjects, card.name);
 
                 const cardGroupInstances = <CardGroupEntity[]>(card instanceof SimpleCard ? [ card ] : card.groups);
-                cardGroupInstances.forEach((cardGroupInstance: CardGroupEntity) => {
+                cardGroupInstances?.forEach((cardGroupInstance: CardGroupEntity) => {
```

Comment:
Can you please also add this question mark `?` to the other occurrences of `cardGroupInstances`. i.e line 86 and 195

Additionally, is it possible to have a card without groups?